Makefile

```diff
@@ -56,15 +60,16 @@ all: build test
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildTag "$(shell git describe --dirty)"
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildTime "$(shell date -u '+%Y/%m/%d %H:%M:%S')"
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildDeps "$(shell GOPATH=${GOPATH} build/depvers.sh)"
+build: TAGS += release
 build:
-	$(GO) build $(GOFLAGS) -ldflags '$(LDFLAGS)' -v -i -o cockroach
+	$(GO) build -tags '$(TAGS)' $(GOFLAGS) -ldflags '$(LDFLAGS)' -v -i -o cockroach
```

Comment:
I often run `go test` directly. Seems like this will cause rebuilds and the potential for subtle differences unless I specify `-tags netgo`. Can you give some more justification for why you want to always specify `-tags netgo`?